lua/nerdy/config.lua

```diff
@@ -4,10 +4,12 @@ local M = {}
 ---@field max_recents integer : Max number of recent icons to keep
 ---@field add_default_keybindings boolean : Whether to add default keybindings
 ---@field use_new_command boolean : Whether to use new Nerdy command
+---@field selection_to_clipboard boolean : Whether to copy selected icon to clipboard
 M.config = {
     max_recents = 30,
     add_default_keybindings = true,
     use_new_command = false,
+    selection_to_clipboard = false,
```

Comment:
```suggestion
    copy_to_clipboard = false,
```

Can we rename the config option please